storage/watchable_store_test.go

```diff
@@ -222,3 +239,109 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestNewMapWatchingToEventMap(t *testing.T) {
+	k0, k1, k2 := []byte("foo0"), []byte("foo1"), []byte("foo2")
+	v0, v1, v2 := []byte("bar0"), []byte("bar1"), []byte("bar2")
+
+	ws := []*watching{
```

Comment:
we can also make this one line. The general rule is to keep things compacted as long as they are still clear. That can improve the readability of the code.